packages/code-du-travail-frontend/src/outils/common/InfosCCn.tsx

```diff
@@ -117,7 +117,7 @@ function StepInfoCCn({
     <>
       <Field
         name={CONVENTION_NAME}
-        validate={isOptional ? null : required}
+        validate={isOptional ? undefined : required}
```

Comment:
```suggestion
isOptional && required
```
Je suis pas sûr que ça fonctionne mais ça me semblerait plus élégant

Reply:
Je trouve ça plus compliqué à lire les `&&` que les conditions. D'ailleurs on se demande si ça marche alors que le if/else la question ne se pose pas.